packages/pds/src/db/migrations/20230718T170914772Z-sequencer-leader-sequence.ts

```diff
@@ -0,0 +1,25 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<any>, dialect: Dialect): Promise<void> {
+  if (dialect === 'sqlite') return
+  const res = await db
+    .selectFrom('repo_seq')
+    .select('seq')
+    .where('seq', 'is not', null)
+    .orderBy('seq', 'desc')
+    .limit(1)
+    .executeTakeFirst()
+  const startAt = res?.seq ? res.seq + 10000 : 1
```

Comment:
I think that 10k works for a dedicated sequencer using the quick-reload, but might not have enough wiggle room for a rolling update.  During normal operations 10k events might occur in around 4mins, being slightly conservative.

Reply:
(So we could just do it with a quick-reload, or up this number.)

Reply:
wow yeah just did the math and... events are coming out fast lol

i'll bump it up to 50k

Reply:
really no downside to overshooting a bit